content/src/ports/failedDeployments.ts

```diff
@@ -0,0 +1,78 @@
+import { AuthChain } from '@dcl/crypto'
+import { EntityType } from '@dcl/schemas'
+import {
+  deleteFailedDeployment,
+  getFailedDeployments,
+  saveFailedDeployment
+} from '../logic/database-queries/failed-deployments-queries'
+import { AppComponents } from '../types'
+
+export enum FailureReason {
+  DEPLOYMENT_ERROR = 'Deployment error' // During sync, there was an error during deployment. Could be due to a validation
```

Comment:
do we need this? It seems a little bit redundant as failed deployments only came from Sync

Reply:
just copied from last implementation